src/libs/nunit.framework.license.txt

```diff
@@ -0,0 +1,15 @@
+Copyright � 2002-2008 Charlie Poole
```

Comment:
This would only be temporary. I figured first step is building, then we upgrade the project files and use NuGet.

Reply:
Only problem is binaries are left in history forever, so we always clone their size around.

Reply:
@jonorossi, @phillip-haydon - Agree with both of you. How about dealing with NUnit in a separate PR (so that we don't have to mix up unrelated tasks) and merge it before this PR?

Reply:
@jonorossi: Also, I am seeing that this project still uses the old (non-SDK) `.csproj` format. If we wanted to pull NUnit from NuGet, would we want to go straight to SDK-style `<PackageReference>` (which would mean the project file must be converted first), or would a `packages.config` be legitimate for the time being (meaning the project file can be converted later)?

Reply:
Yep, as I mentioned in the issue it is a VS2010 project, I'm happy for it to be straight into a new VS2017 SDK style format. I don't mind if the pull requests are a little big.

Reply:
Happy to go straight to VS2017 before converting the project types. 

I’ll update this PR soon.